src/Compilers/CSharp/Test/Semantic/Semantics/DeconstructionTests.cs

```diff
@@ -822,6 +1257,54 @@ static void Main()
             CompileAndVerify(comp, expectedOutput: "1 2");
         }
 
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TupleWithUseSiteError_IOperation()
```

Comment:
> This is a clone of the above test TupleWithUseSiteError, and yes there doesn't seem to be any use-site diagnostic in that test as well. I will rename both the tests.

The ValueTuple type is missing Item2. I guess, the test is ensuring no error is reported because of that, corresponding symbol has use-site diagnostics in this case.

---
In reply to: [144329686](https://github.com/dotnet/roslyn/pull/22479#discussion_r144329686) [](ancestors = 144329686)